.github/workflows/iroha2-dev-pr.yml

```diff
@@ -239,3 +235,61 @@ jobs:
         run: |
           rm -f ~/.gradle/caches/modules-2/modules-2.lock
           rm -f ~/.gradle/caches/modules-2/gc.properties
+
+  coverage:
```

Comment:
It is here just for testing, right?

Reply:
About coverage I am checking how the codecov bot works, it's said it generates nice reports for PRs, if so then I think we can have it in our PR workflow

Reply:
Actually I am thinking that for our push workflow only archive and publish should be there